base/REPL.jl

```diff
@@ -958,7 +956,7 @@ type StreamREPL <: AbstractREPL
     waserror::Bool
     StreamREPL(stream,pc,ic,ac) = new(stream,pc,ic,ac,false)
 end
-StreamREPL(stream::IO) = StreamREPL(stream, julia_green, Base.input_color(), Base.answer_color())
+StreamREPL(stream::IO) = StreamREPL(stream, Base.text_color[:green], Base.input_color(), Base.answer_color())
```

Comment:
The bold is still being applied, just somewhere else, so the color isn't actually changing (nor the bold). I see what you mean though, so I will updated it. It probably makes sense to do the same in the banner too, right?